README.md

```diff
@@ -135,6 +136,49 @@ And then you chain everything together and it will just work :)
 UserQuery.new.where.not(name: nil).active_last_week.not_deleted.order(:id)
 ```
 
+## Generators
+
+To create the query object we can use the rails generator tool. For that, we just run:
+
+    $ rails generate query User
+
+And it will create your UserQuery object at `app/queries` folder. If you have set Rspec as your test framework, the corresponding spec file will be also created at `spec/queries`.
+
+:warning: Rspec is the only test framework supported for now.
+
+If your query object is based on another class, you can set the `associated_relation` attribute to automatically override the `associated_relation` method.
+
+    $ rails generate query CustomUser assoacited_to:User
```

Comment:
```suggestion
    $ rails generate query CustomUser associated_to:User
```

Reply:
shouldn't this be `associated_relation` actually?